### PR TITLE
Updated operator.cc

### PR DESCRIPTION
**Pull request by amitsrivastava78**
Fixed the warnings in the operator.cc file

### CODE REVIEW

**Comment by rthadur** [comment]
@amitsrivastava78  please club related PR's #26193 ,#26192,#26188

**Comment by amitsrivastava78** [comment]
@rthadur  I have merged the PRs as requested by you.

**Comment by rthadur** [comment]
> @rthadur I have merged the PRs as requested by you.

thank you @amitsrivastava78

File: tensorflow/lite/toco/tooling_util.cc [hunk 1]
```diff
@@ -2327,7 +2327,7 @@ void UseArraysExtraInfo(Model* model, bool quantize_output) {
         // Make sure to create the shape even if there are no dims, to
         // correctly record 0-D shapes.
         array.mutable_shape();
-        for (int dim : entry.shape().dims()) {
+        for (const auto & dim : entry.shape().dims()) {
```
Comment (srjoglekar246) [inline tensorflow/lite/toco/tooling_util.cc]: nit: `const auto&`

**Review by srjoglekar246 — CHANGES_REQUESTED** [review]

File: tensorflow/lite/toco/tooling_util.cc (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/lite/toco/tooling_util.cc]: thanks @srjoglekar246 , i have updated the code as per your comments, kindly check and approve.

**Comment by amitsrivastava78** [comment]
thanks @srjoglekar246 , i have updated the code as per your comments, kindly check and approve.

**Comment by amitsrivastava78** [comment]
@srjoglekar246 , i have updated the code as per the comments, kindly check and approve.

**Review by srjoglekar246 — APPROVED** [review]

**Pull request merged**